flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js

```diff
@@ -738,6 +738,20 @@ class CDUFlightPlanPage {
         }
         const allowScroll = waypointsAndMarkers.length > 4;
         if (allowScroll) {//scroll only if there are more than 5 points
+            //Only need to call onAirport()if there are > waypoints
+        //since both origin and destination are already displayed on the same page if no
+            mcdu.onAirport = () => {
+                const isOnFlightPlanPage = mcdu.page.Current === mcdu.page.FlightPlanPage;
+                const destinationAirportOffset = waypointsAndMarkers.length - 5;
+                const allowCycleToOriginAirport = mcdu.flightPhaseManager.phase === FmgcFlightPhases.PREFLIGHT;
+                if (offset === destinationAirportOffset && allowCycleToOriginAirport && isOnFlightPlanPage) {//only show origin if still on ground
+                    offset = 0; // Go back to top of flight plan page to show origin airport.
+                } else {
+                    offset = destinationAirportOffset;
+                }
```

Comment:
```suggestion
                offset = offset === destinationAirportOffset && allowCycleToOriginAirport && isOnFlightPlanPage ? 0 : destinationAirportOffset;
```

Reply:
Is a ternary operator a convention of FBW?  I generally prefer how traditional IF/ELSEs read but if required per FBW style conventions am happy to change to the suggestion.

Reply:
The if is better here. The ternary is too long for one line. You should fix your comments though (put them on the line above where they're more than a couple of words, and make sure you have spaces after the comment operator).

Reply:
ok, will do